### PR TITLE
podman: 3.0.0 -> 3.0.1

### PR DESCRIPTION
**Pull request by zowoq**
https://github.com/containers/podman/releases/tag/v3.0.1

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 113662` at 92f9b27c run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>podman</li>
    <li><a href="https://gist.github.com/235daa4fa406c43d137548537735ab0d">podman-unwrapped</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-patch-comment.md">missing-patch-comment</a>

Please add a comment on the line above, explaining the purpose of this patch.
Near pkgs/applications/virtualization/podman/default.nix:29:5:
```
   |
29 |     ./remove-unconfigured-runtime-warn.patch
   |     ^
```
</li>
  </ul>
</details>

File: pkgs/applications/virtualization/podman/default.nix [hunk 1]
```diff
@@ -16,13 +16,13 @@
...
   pname = "podman";
-  version = "3.0.0";
+  version = "3.0.1";
 
   src = fetchFromGitHub {
     owner = "containers";
     repo = "podman";
     rev = "v${version}";
-    sha256 = "1dsriw2vjzjaddxdhl3wbj2ppnsyi29f4bjwc8lzyz20wfwx4ay4";
+    sha256 = "sha256-+z28Z0KvaJa32+eTGFsNX8g/WVd3BohKoBnNPU/kpWM=";
```
Comment (bhipple) [inline pkgs/applications/virtualization/podman/default.nix:25]:
```suggestion
    hash = "sha256-+z28Z0KvaJa32+eTGFsNX8g/WVd3BohKoBnNPU/kpWM=";
```

**Review by bhipple — CHANGES_REQUESTED** [review]

File: pkgs/applications/virtualization/podman/default.nix (hunk 1 above)
Comment (zowoq) [reply pkgs/applications/virtualization/podman/default.nix:25]: I though we we're waiting for something to happen before doing `sha256` -> `hash`?

Comment (adisbladis) [reply pkgs/applications/virtualization/podman/default.nix:25]: We already have >1400 SRI hashes in nixpkgs.

Comment (adisbladis) [reply pkgs/applications/virtualization/podman/default.nix:25]: There are about 1100 occurrences of `sha256 = "sha256-` in nixpkgs, and I do seem to recall something similar to what @zowoq is saying.

I suggest a treewide PR when the time is right.

Comment (bhipple) [reply pkgs/applications/virtualization/podman/default.nix:25]: I was thinking of updating the nixpkgs-update bot code to make it start fixing this and sending a treewide PR.  Curious if there's a status update for the state of SRI hashes on discourse / github issue somewhere?  Obviously can wait if there's a reason, but it feels really ugly to have n+1 ways of doing things in the meantime!

**Pull request merged**